barretenberg/cpp/src/barretenberg/ecc/scalar_multiplication/scalar_multiplication.hpp

```diff
@@ -157,10 +157,27 @@ typename Curve::Element pippenger(std::span<const typename Curve::ScalarField> s
                                   pippenger_runtime_state<Curve>& state,
                                   bool handle_edge_cases = true);
 
+template <typename Curve>
+typename Curve::Element pippenger(std::span<const typename Curve::ScalarField> scalars,
+                                  std::span<typename Curve::AffineElement> points,
+                                  pippenger_runtime_state<Curve>& state,
+                                  bool handle_edge_cases = true)
+{
+    ASSERT(scalars.size() * 2 <= points.size());
+    return pippenger(scalars, &points[0], state, handle_edge_cases);
+}
 template <typename Curve>
 typename Curve::Element pippenger_unsafe(std::span<const typename Curve::ScalarField> scalars,
                                          typename Curve::AffineElement* points,
                                          pippenger_runtime_state<Curve>& state);
+template <typename Curve>
+typename Curve::Element pippenger_unsafe(std::span<const typename Curve::ScalarField> scalars,
```

Comment:
A future improvement would just get rid of the point-taking versions (and also make sure these spans are const)

Reply:
oh I think this answers my question. I guess they're still used somewhere?